packages/core/src/common/styles.ts

```diff
@@ -9,6 +9,7 @@ export interface Theme {
     textMedium: string;
     textLight: string;
     textBubble: string;
+    textAlign?: "left" | "right" | "center" | "start" | "end";
```

Comment:
I dont think this should be on the theme but a part of BaseCell. Reason for that is theme overrides are rather expensive. Props on base cell are cheap.

Reply:
Currently, the `drawTextCell` does not have access to the cell props, or? Should I pass the info to the `drawTextCell`, `prepTextCell` or handle it similar to how `cell.style` is handled here: https://github.com/glideapps/glide-data-grid/blob/f3f3be6d0aeb4e06fce47d51c44f724ff525f348/packages/core/src/data-grid/data-grid-render.tsx#L1237

Reply:
pass it to the `drawTextCell`, do not pass it to `prepTextCell`

Reply:
Two questions: 1) Is the theme merge also expensive if it is applied on the column instead of every individual cell? 2) If this is a cell prop, we could also make it slightly more generic and call it `contentAlign` so other cell types can also theoretically use it for aligning the content. Would you prefer `textAlign` or `contentAlign`?